lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -22,6 +22,8 @@
 from xmodule.modulestore.tests.factories import CourseFactory
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 
+from edx_notifications.stores.sql.models import SQLNotificationMessage
```

Comment:
@chrisndodge I have removed the store provider import and used the consumer api endpoint get_notifications_for_user and it solves the problem.